examples/hybrid/callbacks.jl

```diff
@@ -13,6 +13,15 @@ function get_callbacks(parsed_args, simulation, model_spec, params)
         save_positions = (false, false),
     )
 
+    additional_callbacks = if !isnothing(model_spec.turbconv_model)
```

Comment:
```suggestion
    tc_callbacks = if !isnothing(model_spec.turbconv_model)
```

Reply:
And below use

`additional_callbacks = (additional_callbacks..., tc_callbacks...)`